src/test/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/AnalysisDetailsTest.java

```diff
@@ -66,6 +67,11 @@
 
 public class AnalysisDetailsTest {
 
+    @BeforeClass
+    public static void setupLocale(){
+        Locale.setDefault(Locale.ENGLISH);
```

Comment:
Which tests fail when you don't set this? We shouldn't be depending on the system default locale in the code, so a failing test would indicate that we're not passing a locale at some point where we should be

Reply:
testCreateAnalysisSummary, testCreateAnalysisSummary2, testCreateAnalysisSummary3, testCreateAnalysisSummary4

Cause is line 131 in AnalysisDetails. It created a new DecimalFormat, and that uses the default locale internally.

(the test failures were already there before this PR)

Reply:
Does changing line 131 to `NumberFormat decimalFormat = new DecimalFormat("#0.00", DecimalFormatSymbols.getInstance(Locale.ENGLISH));` prevent the need for setting System Locale for you?

Reply:
that helps for testCreateAnalysisSummary2, testCreateAnalysisSummary3, testCreateAnalysisSummary4, but fails for testCreateAnalysisSummary...

I don't see why though...

Reply:
ahh found it, there was anotherone in line 313